src/Sylius/Behat/Service/SessionService.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Service;
+
+use Behat\Mink\Mink;
+
+final class SessionService implements SessionServiceInterface
```

Comment:
`*Service` is such a lazy name 😄 Even `SessionManager` would be better

Reply:
Naming things is difficult.